ddtrace/internal/telemetry/writer.py

```diff
@@ -44,6 +45,19 @@
 log = get_logger(__name__)
 
 
+class LogData(dict):
+    def __hash__(self):
+        return hash(tuple([self["message"], self["level"], self.get("tags"), self.get("stack_trace")]))
```

Comment:
```suggestion
        return hash((self["message"], self["level"], self.get("tags"), self.get("stack_trace")))
```
no need to create a list, you can create a tuple directly